products/ASC.Files/Client/src/pages/Home/Section/Body/TableView/sub-components/DateCell.js

```diff
@@ -1,8 +1,11 @@
 import React from "react";
 import { StyledText } from "./CellStyles";
+import moment from "moment";
 
 const DateCell = ({ create, updatedDate, createdDate, sideColor }) => {
-  const date = create ? createdDate : updatedDate;
+  const date = moment(new Date(create ? createdDate : updatedDate)).format(
```

Comment:
As in comment above. Data from "getTableStatusByDate"

Reply:
Removed momentJS from DateCell component.